CHANGELOG.md

```diff
@@ -7,6 +7,7 @@
 - Minor: Received Twitch messages now use the exact same timestamp (obtained from Twitch's server) for every Chatterino user instead of assuming message timestamp on client's side. (#3021)
 - Minor: Received IRC messages use `time` message tag for timestamp if it's available. (#3021)
 - Minor: Added informative messages for recent-messages API's errors. (#3029)
+- Minor: Added "About Chatterino" to About tab. (#2965, #3066)
```

Comment:
Could you add a small description of what's in the About Chatterino section? e.g. that it links to the wiki